11_FFTBloom/app_resources/kernel_fft_first_axis.hlsl

```diff
@@ -0,0 +1,154 @@
+#include "common.hlsl"
+
+/*
+ * Remember we have these defines:
+ * _NBL_HLSL_WORKGROUP_SIZE_
+ * ELEMENTS_PER_THREAD
+ * KERNEL_SCALE
+ * scalar_t
+ * FORMAT
+*/
+
+// TODOS:
+//        - You can get away with saving only half of the kernel (didn't do it here), especially if FFT of the image is always done in the same order (in that case you can just
+//          store the same half of the kernel spectrum as you do the image's).
```

Comment:
by studying the code this TODO seems stale

Reply:
?

Reply:
Done